File: 250721/특정 조건 두 정수 비교/specific-comparison-of-two-natural-numbers.py
# 입력
intA, intB = map(int, input().split())

# 변수 정의
intCase1 = 0;
intCase2 = 0;

# 조건문
if intA < intB :
    intCase1 = 1;
else : 
    intCase1 = 0;

if intA == intB :
    intCase2 = 1;
else :
    intCase2 = 0;

# 출력
print(intCase1, intCase2, sep = " ")

File: 250721/특정 조건 두 정수 비교/specific-comparison-of-two-natural-numbers.java
// import
import java.util.Scanner;

public class Main {
    public static void main(String[] args) {
        // 입력
        Scanner sc = new Scanner(System.in);
        int intA = sc.nextInt();
        int intB = sc.nextInt();

        // 조건문
        if (intA < intB) {
            System.out.print("1 ");
        } else {
            System.out.print("0 ");
        }

        if (intA == intB) {
            System.out.print("1 ");
        } else {
            System.out.print("0 ");
        }
    }
}